Makefile

```
@@ -1,20 +1,30 @@
-#!/usr/bin/env bash
+.PHONY: all clean build swagger test lint run fmt docker-build help
+.DEFAULT_GOAL:=help
 
-.PHONY: all clean build swagger test lint run help
-.DEFAULT_GOAL := help
-
-VERSION := $(shell git describe --always --tags)
-BUILD_TIME=$(shell date '+%Y%m%d-%H%M%S')
+VERSION?=$(shell git describe --always --tags)
+BUILD_TIME?=$(shell date '+%Y%m%d-%H%M%S')
```

To go with the above comment:
```suggestion
BUILD_TIME?=$(shell date '+%Y%m%d-%H%M%S')
GOARCH?=$(shell go env GOARCH)
GOARM?=$(shell go env GOARM)
```

---

Sorry for my previous comment.
I get the problem....a few minutes to late 😓